client/components/Form.jsx

```diff
@@ -0,0 +1,188 @@
+import React from "react";
+import Input from "./Input.jsx";
+import Email from "./Email.jsx";
+import Select from "./Select.jsx";
+import Checkbox from "./Checkbox.jsx";
+import DateBox from "./DateBox.jsx";
+import SubmitButton from "./SubmitButton.jsx";
+import axios from "axios";
+
+class Form extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleFirst = this.handleFirst.bind(this);
+    this.handleLast = this.handleLast.bind(this);
+    this.handleEmail = this.handleEmail.bind(this);
+    this.handlePhone = this.handlePhone.bind(this);
+    this.handleClass = this.handleClass.bind(this);
+    this.handleStatus = this.handleStatus.bind(this);
+    this.handleStart = this.handleStart.bind(this);
+    this.handleFormSubmit = this.handleFormSubmit.bind(this);
+
+    this.state = {
+      newVolunteer: {
+        first_name: "",
+        last_name: "",
+        email_address: "",
+        phone_number: "",
+        class: "Probationary",
+        start_date: 0,
+        end_date: 0,
+        is_admin: false,
+        is_volunteer: false
+      },
+      classes: ["Probationary", "Trainee", "Apprentice", "General"],
+      startDate: new Date()
+    };
+  }
+  componentDidMount () {
+    this.handleStart(new Date())
+  }
+
+  handleFirst(e) {
```

Comment:
We can make these dynamic where we don't have to have a function for each element. 
https://reactjs.org/docs/forms.html#handling-multiple-inputs